dex/src/main/java/io/crate/data/AsyncCompositeBatchIterator.java

```diff
@@ -103,6 +104,7 @@ public void close() {
         }
     }
 
+    @SuppressForbidden
```

Comment:
It would probably make sense to use the new utility function here. Looking at the code it also seems to suffer from the lack of exception handling (loadNextBatch is even documented to *not* throw exceptions)